map/views.py

```diff
@@ -1289,6 +1289,58 @@ def get_tile(self, req, ver, zoom, x, y,
     
         if get_images:
             return rimgs
+
+        if "lslga" in req.GET:
+
+            from PIL import Image, ImageDraw
+            img = Image.open(tilefn)
+
+            ra, dec = wcs.crval
+            pixscale = np.abs(wcs.cd[0]) * 3600
```

Comment:
can go
```
pixscale = wcs.pixel_scale()
```
which should be in arcsec per pixel.

Reply:
OK, thanks.